documentation/definition-of-done.md

```diff
@@ -1,20 +1,16 @@
-# Definition of Done:
+# Definition of Done
 
-- Step by step testing instructions
-- At least one peer review acceptance
-- Chromatic testing by design
-- Definition of done checklist from issue template (eventually)
-- If the route to a live page has been changed, a Permanent Redirect (301) has also been added to the nginx conf file
-- Build pipeline successful, including:
-  - JavaScript linting
+- Step-by-step testing instructions
+- At least one peer review approval
+- If the route to a live page has been changed, a Permanent Redirect (301) has also been added to the `nginx` conf files
```

Comment:
Hmm, this came up recently.  It's easy and useful for static URLs but for URLs with a dynamic component, not so much.

Reply:
Also, this should probably be part of the acceptance criteria, not implied in the DoD.